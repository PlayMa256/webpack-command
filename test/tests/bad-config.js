const execa = require('execa');

const { test } = require('../util');

test('Zero Config', module, () => {
  it('should run', () => {
    const cliPath = resolve(__dirname, '../../lib/cli');
    const cwd = resolve(__dirname, '../fixtures/bad-config');
    const stub = () => execa.sync('node', [cliPath], { cwd });

    expect(stub).toThrowErrorMatchingSnapshot();
  });
});
