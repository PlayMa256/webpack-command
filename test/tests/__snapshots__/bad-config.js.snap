// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Zero Config > should run #0 1`] = `
"Command failed: node <PROJECT_ROOT>/lib/cli
ValidationError: config-loader

  Options Validation Error

  options['batman']  is an invalid additional property

    at validate (<PROJECT_ROOT>/node_modules/@webpack-contrib/schema-utils/dist/validate-options.js:87:15)
    at validator (<PROJECT_ROOT>/node_modules/@webpack-contrib/schema-utils/dist/validate-options.js:118:10)
    at resolve.then (<PROJECT_ROOT>/node_modules/@webpack-contrib/config-loader/lib/index.js:16:7)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:684:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:613:3

"
`;
